static/templates/actors/familiar-sheet.html

```diff
@@ -213,7 +213,8 @@ <h4>{{localize "PF2E.EffectsLabel"}}</h4>
                     <ol class="effects-list item-list">
                         {{#each items as |item idx|}}
                         {{#if (eq item.type "effect")}}
-                            <li class="item effects" data-item-id="{{item._id}}">
+                        {{#if (or (not item.system.secret) @root.user.isGM)}}
+                            <li class="item effects {{#if item.system.secret}}secret{{/if}}" data-item-id="{{item._id}}">
```

Comment:
Does {{#if item.secret}} work? I believe it should. If not that's definitely going to be annoying.

Reply:
For whatever reason, it does not. The getter doesn't work, and if I do `{{log item}}` it clearly shows that it does not exist.

Familiar sheets, am I right.

Reply:
If its only like this in familiar sheets then feel free to ignore it. I'll need to go over this anyways when I work on familiars.